pkg/util/node/node.go

```diff
@@ -144,6 +144,36 @@ func GetNodeHostIP(node *v1.Node) (net.IP, error) {
 	return ips[0], nil
 }
 
+// GetNodeIPs returns IPs (as with GetNodeHostIPs) for the node with the provided name.
+// If required, it will wait for the node to be created.
```

Comment:
I had this in mind, so it is coherent with   `GetNodeHostIP` , just FYI 

https://github.com/kubernetes/kubernetes/pull/99066/commits/eadd0c9fe94834f7e100760f269437fcf1051e2f

Reply:
went a different direction that was closer to linux